src/main/java/org/zalando/nakadi/util/NakadiUtils.java

```diff
@@ -0,0 +1,26 @@
+package org.zalando.nakadi.util;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class NakadiUtils {
```

Comment:
Nakadi**Collection**Utils? :)
Or maybe the method name can be improved to reflect that it's about collections.